packages/components/tests/dummy/app/templates/components/reveal.hbs

```diff
@@ -0,0 +1,167 @@
+            <b>TXT</b>.
+          </li>
+          <li>
+            In the
+            <b>Name/Host/Alias</b>
+            field, enter
+            <b>@</b>
+            or leave blank. Your host might require you to enter your domain, which looks like
+            <b>example.com</b>
+          </li>
+          <li>
+            In the
+            <b>Time to Live (TTL)</b>
+            field, enter
+            <b>86400</b>
+            or leave the default.
+          </li>
+          <li>
+            In the
+            <b>Value/Answer/Destination</b>
+            field, paste the verification record you copied.
+          </li>
+          <li>
+            Save the record.
+          </li>
+        </ol>
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With interactive content">
+      <Hds::Reveal @text="More options">
+        See here for more info:
+        <a href="https://www.hashicorp.com/" target="_blank" rel="noopener noreferrer">HashiCorp</a>
+      </Hds::Reveal>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H3>Used next to other content</Shw::Text::H3>
+
+  <Shw::Flex @direction="column" {{style gap="1rem"}} as |SF|>
+    <SF.Item @label="Used beneath a form input">
+      <Hds::Form::TextInput::Field as |F|>
+        <F.Label>Private subnet 1</F.Label>
+        <F.HelperText>Ensure that the private subnets have access to the external network.
+        </F.HelperText>
+      </Hds::Form::TextInput::Field>
+      <Hds::Reveal @text="Where can I find this?">
+        <Hds::Alert @type="compact" as |A|>
+          <A.Description>Navigate to Subnets(1), if you already have subnet created for the VPC of choice, copy the
+            Subnet ID(2). Otherwise, Create subnet(3) for chosen VPC.</A.Description>
+        </Hds::Alert>
+        <p>
+          <img
+            src="https://portal.cloud.hashicorp.com/engines-dist/consul/images/aws-subnet-instruction-7c866fcfb753c4794ff07afe5523818c.png"
+            alt="example"
+          />
+        </p>
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With text above and below">
+      <p class="hds-typography-body-300">
+        Lorem ipsum dolor sit amet consectetur
+      </p>
+      <Hds::Reveal @text="More options">
+        Additional content
+      </Hds::Reveal>
+      <p class="hds-typography-body-300">
+        adipisicing elit. Doloremque blanditiis sapiente iste beatae voluptates voluptatum.
+      </p>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H4>Edge case</Shw::Text::H4>
+
+  <Shw::Flex @direction="column" {{style gap="1rem"}} as |SF|>
+    <SF.Item @label="Used inline with text">
+      <p class="hds-typography-body-300">
+        Lorem ipsum dolor sit amet consectetur
+        <Hds::Reveal @text="More options">
+          Additional content
+        </Hds::Reveal>
+        adipisicing elit. Doloremque blanditiis sapiente iste beatae voluptates voluptatum.
+      </p>
+    </SF.Item>
+  </Shw::Flex>
```

Comment:
@KristinLBradley I see that this says "Edge case," but has there been an example of this somewhere? If so, this makes me worried about our approach to the overhanging states because they end up covering the text when hovered. When opened, it also pushes the original content down, which is quite an odd experience. I would expect the original content to remain in place and the additional (hidden) content to open below.  

cc @cveigt

Reply:
This is not a real use case on our product. But even if it was, I would expect the behavior to be as @heatherlarsen  describes 

> I would expect the original content to remain in place and the additional (hidden) content to open below.

One thing to note is that the showcase is an internal tool (at least for now) and is used to test different scenarios. We should not expect this to be anywhere on the documentation website.